libbpf-cargo/src/build.rs

```diff
@@ -165,6 +166,18 @@ fn check_progs(progs: &[UnprocessedProg]) -> Result<()> {
     Ok(())
 }
 
+fn extract_version(output: &str) -> Result<&str> {
+    let re = Regex::new(r"clang\s+version\s+(?P<version_str>\d+\.\d+\.\d+).*")?;
```

Comment:
maybe add beginning of string matching (`^`)? and .* is probably not necessary, given this is a substring match

Reply:
Can't add `^` b/c the ubuntu format adds "Ubuntu" to the front